packages/frontend-core/src/api/attachments.js

```diff
@@ -88,5 +88,19 @@ export const buildAttachmentEndpoints = API => {
         },
       })
     },
+
+    /**
+     * Download an attachment from a row given its column name.
+     * @param tableId
```

Comment:
Probably should fill out descriptions for these to be inline with the other functions here